compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEDmaUtils.cpp

```diff
@@ -39,6 +64,27 @@ bool areAccessPatternsCombinable(const SmallVector<OpFoldResult> &offsetsA,
   if (offsetsA.size() == offsetsB.size() && offsetsA.size() + 1 > maxNbDims)
     return false;
 
+  // Equality of the last N elements of the access patterns of A and B with N =
+  // abs(sizeB - sizeA) results in some simple cases in which the access
```

Comment:
```suggestion
  // min(sizeA, sizeB) results in some simple cases in which the access
```
maybe?

Reply:
Not sure what you mean? `abs(sizeB - sizeA)` is not the same as `min(sizeA, sizeB)`?

Reply:
is sizeA the rank, i.e. offsetA.size() ? So if sizeA = sizeB = 4, then you're checking no equality (N = 0)?